src/components/PlaidLink/index.native.js

```diff
@@ -1,55 +1,46 @@
-import React from 'react';
+import {useEffect} from 'react';
 import {NativeEventEmitter} from 'react-native';
-import {openLink} from 'react-native-plaid-link-sdk';
+import {openLink, useDeepLinkRedirector} from 'react-native-plaid-link-sdk';
 import Log from '../../libs/Log';
 import CONST from '../../CONST';
 import nativeModule from './nativeModule';
 import {plaidLinkPropTypes, plaidLinkDefaultProps} from './plaidLinkPropTypes';
 
-class PlaidLink extends React.Component {
-    constructor(props) {
-        super(props);
-        this.listener = undefined;
-    }
 
-    componentDidMount() {
+const PlaidLink = (props) => {
+    let listener;
+    useDeepLinkRedirector();
+    useEffect(() => {
         const emitter = new NativeEventEmitter(nativeModule);
-        this.listener = emitter.addListener('onEvent', this.onEvent.bind(this));
+        listener = emitter.addListener('onEvent', (event) => {
```

Comment:
This looks like it will work, but I noticed there's also this in the docs

https://github.com/plaid/react-native-plaid-link-sdk#to-receive-onevent-callbacks